configs/11.0/packages/valgrind/sources

```diff
@@ -20,11 +20,11 @@
 #
 
 ATSRC_PACKAGE_NAME="Valgrind"
-ATSRC_PACKAGE_VER=3.11.0
+ATSRC_PACKAGE_VER=3.13.0
 # Don't update the Valgrind revision without updating the VEX revision too.
 # They have to be in sync.
-ATSRC_PACKAGE_REV=15905
-ATSRC_PACKAGE_VEX_REV=3223
+ATSRC_PACKAGE_REV=16446
+ATSRC_PACKAGE_VEX_REV=3398
```

Comment:
This is not the correct combination.

Revision 3398 in vex's trunk is newer than 16446 in valgrind's trunk.
Besides that, revision 3392 was not backported to branches/VALGRIND_3_13_BRANCH/VEX.